scripts/thrust_model_v2/Dynamic/generate_crazyflie_thrust_model.py

```diff
@@ -0,0 +1,128 @@
+import math
+import sys
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.interpolate import interp1d
+import scipy.fftpack as fftpack
+from scipy import signal
+from scipy.optimize import lsq_linear
+from mpl_toolkits.mplot3d import Axes3D
+from numpy.polynomial import Polynomial as P
+import yaml
+import datetime
+
```

Comment:
Same complaint as the sim script, these import statements are all over the place and don't follow a nice convention. Like instead of `import scipy.fftpack as fftpack` you can do what you did the line before and say `from spicy import fftpack`